Rubberduck.Parsing/ComReflection/ComParameter.cs

```diff
@@ -12,71 +12,77 @@
 
 namespace Rubberduck.Parsing.ComReflection
 {
-    [DebuggerDisplay("{DeclarationName}")]
+#if DEBUG
+    [DebuggerDisplay("{" + nameof(DeclarationName) + "}")]
+#endif
     public class ComParameter
     {
-        public string Name { get; private set; }
+        public static ComParameter Void = new ComParameter { _typeName = new ComTypeName(null, string.Empty) };
 
-        public string DeclarationName
-        {
-            get
-            {
-                return string.Format("{0}{1} {2} As {3}{4}{5}",
-                    IsOptional ? "Optional " : string.Empty,
-                    IsByRef ? "ByRef" : "ByVal",
-                    Name,
-                    TypeName,
-                    IsOptional && DefaultValue != null ? " = " : string.Empty,
-                    IsOptional && DefaultValue != null ?
-                        IsEnumMember ? DefaultAsEnum : DefaultValue
-                        : string.Empty);
-            }
-        }
+        public string Name { get; }
+
+#if DEBUG
+        public string DeclarationName => string.Format("{0}{1} {2} As {3}{4}{5}",
+            IsOptional ? "Optional " : string.Empty,
+            IsByRef ? "ByRef" : "ByVal",
+            Name,
+            TypeName,
+            IsOptional && DefaultValue != null ? " = " : string.Empty,
+            IsOptional && DefaultValue != null ?
+                _typeName.IsEnumMember ? DefaultAsEnum : DefaultValue
+                : string.Empty);
+#endif
 
         public bool IsArray { get; private set; }
         public bool IsByRef { get; private set; }
         public bool IsOptional { get; }
         public bool IsReturnValue { get; }
         public bool IsParamArray { get; set; }
+        public object DefaultValue { get; }
+
+        public string DefaultAsEnum
+        {
+            get
+            {
+                if (!_typeName.IsEnumMember || !ComProject.KnownEnumerations.TryGetValue(_typeName.EnumGuid, out ComEnumeration enumType))
+                {
+                    return string.Empty;
+                }
+                var member = enumType.Members.FirstOrDefault(m => m.Value == (int)DefaultValue);
```

Comment:
Yes. In a case where there are members with identical values, there isn't a way to discriminate between them. For example, `vbIMEModeOff` and `vbIMEOff` both have a value of 2. Makes no real difference functionally which one gets picked, because they are equivalent.